nav2_behavior_tree/include/nav2_behavior_tree/plugins/action/compute_path_to_pose_action.hpp

```diff
@@ -70,6 +70,8 @@ class ComputePathToPoseAction : public BtActionNode<nav2_msgs::action::ComputePa
     return providedBasicPorts(
       {
         BT::OutputPort<nav_msgs::msg::Path>("path", "Path created by ComputePathToPose node"),
+        BT::OutputPort<nav2_msgs::msg::GlobalPlannerResultCode>("global_planner_result_code",
```

Comment:
Just an int32 perhaps? Or I think you can even do `nav2_msgs::msg::ComputePathToGoal::Response::ErrorCode` (or similar)